qemu/firmware/firmware.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package network provides firmware configurators for use with the Go qemu
```

Comment:
Package firmware (typo)

Sorry to change my mind one more time -- can we make it "qfirmware"? I think I'm going to rename network to qnetwork and introduce e.g. qfiles.

Reply:
Sure, works for me :)